### ISSUE TITLE
error: option --add-lofar-utils not recognized

### ISSUE THREAD

**Issue by AlexKurek** [lofar-astron/RMextract#37]
This is what Im getting while trying to compile curent master:
```
+ mkdir -p /opt/RMextract-0.4/build/
+ cd /opt/RMextract-0.4/build/
+ git clone https://github.com/maaijke/RMextract.git source/
Cloning into 'source'...
+ cd source/
+ mkdir -p /opt/RMextract-0.4/lib/python2.7/site-packages/
+ python ./setup.py build --add-lofar-utils
/usr/lib/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'long_description_content_type'
  warnings.warn(msg)
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: option --add-lofar-utils not recognized
FATAL:   While performing build: while running engine: exit status 1
```

**Comment by AlexKurek** [comment]
196b0d1 does compile.

**Comment by maaijke** [comment]
THe latest build is now pip installable

**Comment by gmloose** [comment]
Fixed by commit [af20e64](https://github.com/lofar-astron/RMextract/commit/af20e64df6719fdddb0fa99fd48761600ef00529).
